docs/sphinx/pom.xml

```diff
@@ -59,7 +59,6 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>exec-maven-plugin</artifactId>
-        <version>1.2.1</version>
```

Comment:
These are inside the eclipse m2 lifecycle-mapping plugin, which I've moved out into a eclipse-specific build profile.  I've updated the versions.

I'm unsure this is used by anyone on the team.  It's only been set up for formats-api and formats-bsd, which makes me think this is maybe scifio-related.  It looks incomplete though.

If I delete all this me2 lifescycle stuff, it still imports into eclipse and builds without trouble, so I'm unsure what specific value this adds.